tests/foreman/ui/test_host.py

```diff
@@ -3146,3 +3146,219 @@ def _cleanup():
+        2. Bulk organization/location change operation succeeds
+        3. Hosts are successfully moved to new organization/location
+        4. Host properties reflect the new taxonomies
+
+    :CaseComponent: Hosts
+
+    :Team: Phoenix-subscriptions
+    """
+    # Create second organization and location for the test
+    second_org = module_target_sat.api.Organization().create()
+    second_location = module_target_sat.api.Location(organization=[second_org]).create()
+
+    # Create 2 hosts
+    host_names = []
+    for _ in range(2):
+        host = module_target_sat.cli_factory.make_fake_host(
+            {
+                'organization': module_org.name,
+                'location': module_location.name,
+            }
+        )
+        host_names.append(host.name)
+
+    @request.addfinalizer
+    def cleanup():
+        second_org.delete()
+        second_location.delete()
+
+    # Verify hosts are initially in the first org/location
+    for host_name in host_names:
+        host_entity = module_target_sat.api.Host().search(query={'search': f'name={host_name}'})[0]
+        assert host_entity.organization.id == module_org.id
+        assert host_entity.location.id == module_location.id
+
+    # Perform bulk organization/location change through UI
+    with module_target_sat.ui_session() as session:
+        # Select the first organization to see the hosts
+        session.organization.select(module_org.name)
+        session.location.select(module_location.name)
+
+        # Scenario 1 - Change organization
+        session.all_hosts.change_associations_organization(
+            host_names=host_names,
+            new_organization=second_org.name,
+        )
+        # Switch to second organization to verify the change
+        session.organization.select(second_org.name)
+
+        # Scenario 2 - Change location
+        session.all_hosts.change_associations_location(
+            host_names=host_names,
+            new_location=second_location.name,
+        )
+        # Switch to second location to verify the change
+        session.location.select(second_location.name)
+
+        # Verify hosts appear in the new organization and new location
+        for host_name in host_names:
+            host_values = session.all_hosts.search(host_name)
+            assert host_values is not None, f"Host {host_name} not found in new organization"
+
+        # API verification - verify hosts are now in the second org/location
+        for host_name in host_names:
+            host_entity = module_target_sat.api.Host().search(
+                query={'search': f'name={host_name}'}
+            )[0]
+            assert host_entity.organization.id == second_org.id, (
+                f"Host {host_name} not moved to second organization"
+            )
+            assert host_entity.location.id == second_location.id, (
+                f"Host {host_name} not moved to second location"
+            )
```

Comment:
Use single quotes wherever possible for the consistency 
```suggestion
            assert host_entity.organization.id == second_org.id, (
                f'Host {host_name} not moved to second organization'
            )
            assert host_entity.location.id == second_location.id, (
                f'Host {host_name} not moved to second location'
            )
```